responsive enhancements (#944)
unoplatform/uno.toolkit.ui: doc/helpers/responsive-extension.md

#else
	MainWindow = Microsoft.UI.Xaml.Window.Current;
#endif

	// ...
	var helper = Uno.Toolkit.UI.ResponsiveHelper.GetForCurrentView();
	helper.HookupEvent(MainWindow);
}
```
## Platform limitation (UWP-desktop)
`ResponsiveExtension` relies on `MarkupExtension.ProvideValue(IXamlServiceProvider)` to find the target control and property for continuous update of value, and to obtain the property type to apply automatic type conversion as its value properties are parsed as string by the xaml engine. Because this overload is a new addition to WinUI only, UWP project targeting windows will not have access to these features. Uno UWP project against non-windows targets do not have this limitation. However, the windows app may crash or misbehave if you attempt to use this markup on non-string property.
```xml
<Border Background="{utu:Responsive Narrow=Red, Wide=Blue}"
        Tag="This will not work on Uwp for windows" />
```

> You can workaround this, by declaring the values are resources and use `{StaticResource}` to refer to them:

Reviewer: ```suggestion
You can workaround this by declaring the values as resources and using {StaticResource} to refer to them:
```
